templates/auth_kerb.conf.erb

```diff
@@ -8,6 +8,20 @@
   KrbAuthRealms <%= @facts['foreman_ipa']['default_realm'] %>
   Krb5KeyTab <%= scope.lookupvar('::foreman::http_keytab') %>
   KrbLocalUserMapping On
+
+  # Set headers for proxy requests
+  RequestHeader unset REMOTE_USER
+  RequestHeader unset REMOTE_USER_EMAIL
+  RequestHeader unset REMOTE_USER_FIRSTNAME
+  RequestHeader unset REMOTE_USER_LASTNAME
+  RequestHeader unset REMOTE_USER_GROUPS
```

Comment:
If i'm not mistaken, this template is only used if the installer is used to set up external auth, while we would want to always clear these headers if they come from the user

Reply:
The only place in foreman code these headers are read is in the sso/apache.rb which goes to extlogin path before reading them, one more place reads REMOTE_USER for what appears to be session management (in app/controllers/application_controller.rb), I have not modified it there and it is not affecting anything during my tests. Since these are not read anywhere else in Foreman, I don't think we have to clear them every time.

Reply:
I think one concern @ekohl raised on the other pr is that someone might turn on the setting on foreman by mistake without configuring apache accordingly. These headers should never originate from the user, so I believe it should be safe to always clear them.
the other place that uses `REMOTE_USER` should also be updated to the new header - it doesn't break anything, but for remote users there is some different handling of session expiry.

Reply:
I have amended the commit to unset them in the main apache configuration for clearing the RequestHeaders. Do you thing there is need to change "REMOTE_USER environment variable" to "REMOTE_USER HTTP header" in settings description and translations?

Reply:
That sounds like a good idea to update the description, don't worry about the translations as they are generated automatically as part of the release process.

Reply:
I have pushed the updated code with updated translation and tests.